changelog/unreleased/features/2719--add-pseudonymize-pipeline-operator.md

```diff
@@ -0,0 +1 @@
+VAST has a new `pseudonymize` pipeline operator that pseudonymizes IP addresses in user-specified fields.
```

Comment:
Please rewrite in active voice (and limit to 80 cols.)

Reply:
Fixed in d80c8d03c